src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1038,6 +1041,11 @@ def random_mac(self):
                         f'{schema_name}.vcpus',
                         'Only one Virtual CPU is allowed in this system.',
                     )
+            elif not flags['intel_vmx'] and not flags['amd_rvi']:
+                verrors.add(
+                    schema_name,
+                    'Virtualization is not supported by this system!'
```

Comment:
Don't shout at the user.
```suggestion
                    'This system does not support virtualization.'
```